oracle_server/oracle_server.go

```diff
@@ -407,7 +408,7 @@ func (os *OracleServer) reportWithoutCommitment(lastRoundData *types.RoundData)
 		os.logger.Error("do report", "error", err.Error())
 		return err
 	}
-	os.logger.Info("report without commitment", "TX hash", tx.Hash(), "Nonce", tx.Nonce())
+	os.logger.Info("reported last round data and without current round commitment", "TX hash", tx.Hash(), "Nonce", tx.Nonce())
```

Comment:
Should this be info, Or Debug?

Reply:
> Should this be info, Or Debug?

It could be info, because this is the last report of an oracle node who is no longer in the committee due to the epoch rotation.